decred/tests/unit/dcr/wire/test_msgtx.py

```diff
@@ -225,155 +499,141 @@ def test_tx_serialize(self):
             [noTx, noTx, noTxEncoded, []],
             # Multiple transactions.
             [
-                msgtx.multiTx(),
-                msgtx.multiTx(),
-                msgtx.multiTxEncoded(),
-                msgtx.multiTxPkScriptLocs,
+                multiTx(),
+                multiTx(),
+                multiTxEncoded(),
+                multiTxPkScriptLocs,
             ],
         ]
 
         for i, (inTx, out, testBuf, pkScriptLocs) in enumerate(tests):
             # Serialize the transaction.
             buf = inTx.serialize()
-            self.assertEqual(len(buf), inTx.serializeSize(), msg="buflen %i" % i)
-            self.assertEqual(buf, testBuf, msg="buf contents %i" % i)
+            assert len(buf) == inTx.serializeSize(), f"buflen {i}"
+            assert buf == testBuf, f"buf contents {i}"
 
             # Deserialize the transaction.
             tx = msgtx.MsgTx.deserialize(testBuf.copy())
-
-            self.assertEqual(tx, out, msg="txs %i" % i)
+            assert tx == out, f"txs {i}"
 
             # Ensure the public key script locations are accurate.
             psl = inTx.pkScriptLocs()
             if pkScriptLocs is None:
-                self.assertEqual(psl, pkScriptLocs, msg="psl none %i" % i)
+                assert psl == pkScriptLocs, f"psl none {i}"
             else:
-                self.assertListEqual(psl, pkScriptLocs, msg="psl %i" % i)
+                assert psl == pkScriptLocs, f"psl {i}"
                 for j, loc in enumerate(psl):
                     wantPkScript = inTx.txOut[j].pkScript
                     gotPkScript = testBuf[loc : loc + len(wantPkScript)]
-                    self.assertEqual(gotPkScript, wantPkScript, msg="scripts %i" % i)
+                    assert gotPkScript == wantPkScript, f"scripts {i}"
 
-    def test_tx_overflow_errors(self):
+    def test_tx_overflow_errors(self, prepareLogger):
         """
-        TestTxOverflowErrors performs tests to ensure deserializing transactions
-        which are intentionally crafted to use large values for the variable number
-        of inputs and outputs are handled properly.  This could otherwise potentially
-        be used as an attack vector.
+        test_tx_overflow_errors performs tests to ensure deserializing
+        transactions which are intentionally crafted to use large values for
+        the variable number of inputs and outputs are handled properly.  This
+        could otherwise potentially be used as an attack vector.
         """
         # Use protocol version 1 and transaction version 1 specifically
         # here instead of the latest values because the test data is using
         # bytes encoded with those versions.
         pver = 1
-        txVer = 1
-        # buf     []byte // Wire encoding
-        # pver    uint32 // Protocol version for wire encoding
-        # version int32  // Transaction version
-        # err     error  // Expected error
```

Comment:
`pver` is always the same and `txVer` is not used at all, removing from test data.